README.md

```diff
@@ -274,6 +274,62 @@ NxtSupport::BirthDate.new(date: '1990-08-08').to_age # => 30
 NxtSupport::BirthDate.new(date: '1990-08-08').to_age_in_months # => 361
 ```
 
+### NxtSupport/Services
+Enjoy your service objects.
+
+#### NxtSupport::Services::Base
+
+`NxtSupport::Services::Base` gives you some convenient sugar for service objects.
+Instead of:
+```ruby
+WeatherFetcher.new(location: 'Heidelberg').call
+```
+
+You can use:
+```ruby
+WeatherFetcher.call(location: 'Heidelberg')
+```
+
+##### Usage
+You have to include `NxtSupport::Services::Base` in your service:
+```ruby
+class WeatherFetcher
+  include NxtSupport::Services::Base
+
+  def call
+    'Getting the weather..'
+  end
+end
+```
+
+Use it:
+```ruby
+WeatherFetcher.call # => 'Getting the weather..'
+```
+
+##### With a custom method name
+If you implement a method other than `#call`, you can use `#class_interface call: :your_method`
+```ruby
+class HomeBuilder
+  include NxtSupport::Services::Base
+  include NxtInit
+
+  attr_init :width, :length, :height, :roof_type
+
+  class_interface call: :build
```

Comment:
Not real sure when/why we want this.

The initial implementation allowed

```ruby
class_interface :call
# or
class_interface :build
# or
class_interface :method1, :method2
```

which I think is simpler and more useful.

```ruby
define_singleton_method config.values.first do |*args, **opts|
  build_instance(*args, **opts).send(config.values.first)
end
```
This ignores the hash key so the hash doesn't carry any information with it.

@getand The ticket specifically mentions the use of hash. What's the intention?

Reply:
```ruby
class HouseBuilder
  include NxtSupport::Services::Base

  class_interface call: :build

  def build; end
end

HouseBuilder.call # instead of HouseBuilder.build
```

Is the intention being able to do something like this?

Reply:
Idea was to be able to map from class level to instance level. But maybe it's even bad to allow that?

Reply:
So yes, like you described in your example. I mean it would allow us to basically still have factories that implement build but have call on factory class level. I was thinking this could be useful for reducing over services. But don't insist to have it.

Reply:
@AkihikoITOH Is it just about the interface or also about functionality?

Reply:
I don't have strong opinion about having the functionality. The interface makes sense I think.

So we now allow defining only one class interface method at a time, which I think is fine because in most cases we just need one anyway.